src/PKSim.Core/Model/Individual.cs

```diff
@@ -85,7 +90,12 @@ public virtual bool IsPreterm
 
       public virtual void AddMolecule(IndividualMolecule molecule) => Add(molecule);
 
-      public virtual void RemoveMolecule(IndividualMolecule molecule) => RemoveChild(molecule);
+      public virtual void RemoveMolecule(IndividualMolecule molecule)
+      {
+         RemoveChild(molecule);
+         var allMoleculeBuildingBlocks = GetAllChildren<IContainer>(x => x.IsNamed(molecule.Name));
```

Comment:
@Yuri05 I don't thinl this is a problem. We may have to also check the container type?

Reply:
Think should be OK. Just would rename the variable from `allMoleculeBuildingBlocks `to `allMoleculeContainers`